doc/classes/FileAccess.xml

```diff
@@ -517,12 +517,14 @@
 		</constant>
 		<constant name="WRITE" value="2" enum="ModeFlags">
 			Opens the file for write operations. The file is created if it does not exist, and truncated if it does.
+			[b]Note:[/b] If creating a file it must be in an already existing directory, to create the parent path recursively, see [method DirAccess.make_dir_recursive]).
```

Comment:
The new note gave a few suggestions:
```suggestion
			[b]Note:[/b] When creating a new file it must be in an directory that already exists. To recursively create directories for a file path, see [method DirAccess.make_dir_recursive]).
```

Reply:
The phrasing is based on that of `DirAccess` and I prefer "an already existing directory", but will adjust some of it